python/tests/core/util/test_vectors.py

```diff
@@ -104,6 +104,34 @@ def test_vector_operations_with_exceptions(self):
         with self.assertRaises(TypeError):
             iv1 *= dv1
 
+    def test_setitem_getitem(self):
+        primes = [2,3,5,7,11,13,17,19]
+        primep = [i in primes for i in range(20)]
+        b = BoolVector(initial_size = 20)
```

Comment:
This `BoolVector` should be initialized with `initial_value = False`. Now I fear you are not really testing the `__setitem__()` functionality.